matrix/Dcm.hpp

```diff
@@ -62,7 +116,15 @@ class Dcm : public Matrix<Type, 3, 3>
         dcm(2, 2) = aSq - bSq - cSq + dSq;
     }
 
-    Dcm(const Euler<Type> & euler) {
+    /**
+     * Set from euler angles
+     *
+     * Instance is set from angle tripplet (3,2,1) representing
```

Comment:
This provides the transformation matrix from frame 2 to frame 1 where the rotation from frame 1 to frame 2 is described by a 3-2-1 intrinsic Tait-Bryan rotation sequence